### DISCUSSION TITLE
Install problem slinger on pixel 8a

### DISCUSSION THREAD

**Discussion by mdehrlich** [GerryDazoo/Slinger#655] [General]
I had no problem with installing on pixel 6a with android 14, but it will not install on my new 8a?  Help

**Comment by GerryDazoo** [comment]
Are you referring to SlingerPlayer? The Pixel 8a only runs 64 bit apps. download the 64 bit version from github

**Comment by mdehrlich** [comment]
Thanks.  I didn't know there was a 64 bit version.  Great job.
